test/e2e/framework/metrics_util.go

```diff
@@ -53,7 +53,7 @@ const (
 
 	// We set a higher threshold for list apicalls as they can take more time when
 	// the list is really big. For eg. list nodes in a 5000-node cluster.
-	apiListCallLatencyThreshold time.Duration = 2 * time.Second
+	apiListCallLatencyThreshold time.Duration = 5 * time.Second
```

Comment:
There is a developer-oriented SLO that is linear (if you carefully read the whole doc). And if we use this one, we would caught the problem you described.

The step one is supposed to be easy to understand, as it is user-oriented. And by design, it's not linear, because the linear one just has to be complicated.